dappnode/DNP_DAPPMANAGER
# Check for rest of the packages for HTTPS portal

## Context

Packages which have HTTPS mappings on update inject docker-compose which doesn't necessarily have external network specified

## Approach

During installation process, DNP_HTTPS is queried whether mapping exists, and if it does it adds external network to the compose. 

## Test instructions

- Do HTTPS mapping on a package
- Update that same package
- Check if mapping is broken after update
- Test specially when package === DNP_HTTPS and when it is not
